sys/base64/Kconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_BASE64
+    bool "Base64 codec"
+    depends on TEST_KCONFIG
+
+config MODULE_BASE64URL
+    bool "Base64 URL codec"
+    depends on TEST_KCONFIG
+    select MODULE_BASE64
```

Comment:
```suggestion
menuconfig MODULE_BASE64
    bool "Base64 codec"
    depends on TEST_KCONFIG

config MODULE_BASE64URL
    bool "Base64 URL codec"
    depends on MODULE_BASE64
```

Reply:
I think it can just select it as `MODULE_BASE64` has not dependencies.